whats_fresh/whats_fresh_api/templates/entry.html

```diff
@@ -3,38 +3,47 @@
 {% block content %}
 <div class='entry'>
     <a href='{% url 'list-vendors-edit' %}'>
-        <h4>Vendors</h4>
+        <!-- <h4>Vendors</h4> -->
+        <h4>{% theme|get_fieldname:'vendors' %}</h4>
```

Comment:
@tschuy What is the correct way to express this? Currently it gives me an error -

`Invalid block tag: 'theme|get_fieldname:'vendors'', expected 'endblock'`